README.md

```diff
@@ -35,3 +35,21 @@ If you wish to manually create and publish a package for Python:
     twine upload --repository-url ${PYPI_REPOSITORY_URL} --username ${PYPI_USER} --password ${PYPI_PASSWORD} dist/*
     ```
     `PYPI_REPOSITORY_URL`, `PYPI_USER` and `PYPI_PASSWORD` are parameters for publishing.
+
+## Release notes
+
+### 0.0.4
+
++ Update serviceGeneratorVersion and grpcCommonVersion to support gRPC pins filters.
+
+### 0.0.3
+
++ Downgrade grpcVersion and protobufVersion for compatibility with th2-grpc-common.
+
+### 0.0.2
+
++ Added 'encode' method
```

Comment:
I think we can squash all of this into `0.0.2` version. And it seems like changes from `0.0.3` version can be thrown out altogether